src/marketing/google-tag-manager-create-tag.md

```diff
@@ -7,65 +7,71 @@ If you have a Google AdWords account, you can create a tag that tracks conversio
 
-    * Conversion ID
-    * Conversion Label
+    - Conversion ID
+    - Conversion Label
 
-    If you need help, visit Google’s [support site][2]{: target="_blank"}.
+    If you need help, visit Google’s [support site][2].
 
-1. From the Google Tag Manager dashboard, tap <span class="btn">Google AdWords</span>. Then, do the following:
+1. From the Google Tag Manager dashboard, click <span class="btn">Google AdWords</span>. Then, do the following:
 
-    * Tap the title placeholder, and enter a name for the new tag.
+    - Click the title placeholder and enter a name for the new tag.
 
-    * Under **Choose Product**, select **Google Adwords**.
+    - Under **Choose Product**, select **Google Adwords**.
 
-        ![]({% link images/images-ee/google-tag-manager-create-adwords-tag1.png %}){: .zoom}
-        *Choose Product*
+        ![Google Tag Manager - choose Google Adwords]({% link images/images-ee/google-tag-manager-create-adwords-tag1.png %}){: .zoom}
+        _Choose Product_
 
-    * Under **Choose a Tag Type**, select **AdWords Conversion Tracking**. Then, tap <span class="btn">Continue</span>.
+    - Under **Choose a Tag Type**, select **AdWords Conversion Tracking**. Then, tap <span class="btn">Continue</span>.
 
-        ![]({% link images/images-ee/google-tag-manager-create-adwords-tag2.png %}){: .zoom}
+        ![Google Tag Manager - Adwords conversion tracking]({% link images/images-ee/google-tag-manager-create-adwords-tag2.png %}){: .zoom}
+        _Adwords Conversion Tracking_
 
-1. Enter the **Conversion ID** and **Conversion Label** from your AdWords account. Then, tap <span class="btn">Continue</span>.
+1. Enter the **Conversion ID** and **Conversion Label** from your AdWords account. Then, click <span class="btn">Continue</span>.
 
-    ![]({% link images/images-ee/google-tag-manager-create-adwords-tag3.png %}){: .zoom}
-    *Configure Tag*
+    ![Google Tag Manager - conversion ID and label]({% link images/images-ee/google-tag-manager-create-adwords-tag3.png %}){: .zoom}
+    _Configure Tag_
 
 ## Step 2. Create a Rule
 
 Continuing from the Google Tag Manager dashboard, the next step is to create a rule that fires the tag on the conversion page.
 
-1. Under **Fire On**, tap <span class="btn">Some Pages</span>.
+1. Under **Fire On**, click <span class="btn">Some Pages</span>.
 
-    ![]({% link images/images-ee/google-tag-manager-create-adwords-tag4.png %}){: .zoom}
+    ![Google Tag Manager - fire on]({% link images/images-ee/google-tag-manager-create-adwords-tag4.png %}){: .zoom}
+    _Fire On Triggers_
 
 1. In the Choose Pages section, complete the following settings:
 
     |Name:|Enter a name for the page description.|
     |Variable:|url|
-    |Operation:|matches RegEx. To learn more, see: What is “matches regex” for? in Tag Manager Help|
+    |Operation:|matches RegEx. (To learn more, see [What is “matches regex” for?][3] in Tag Manager Help.)|
     |Value:|checkout/success.*|
```

Comment:
i don't see a * explanation

Reply:
I think that is a "wildcard" in the RegEx. That is part of their syntax. Think we should probably use code for these.